apps/web/styles/themes.js

```diff
@@ -0,0 +1,101 @@
+/* Venice's Core Theme */
```

Comment:
why is this file `.js` when all others are `.ts`?

Reply:
Compiler seems to barf when importing it into tailwind's config file as ts while also importing into tsx files as a module. You'll have to show me